.pre-commit-hooks.yaml

```diff
@@ -5,7 +5,7 @@
   language: python
   language_version: python3
   pass_filenames: false
-  files: ^pyproject.toml$
+  files: .*pyproject.toml
```

Comment:
Minor, but I think this may match files like `p-pyproject.toml` or `pyproject.toml.old`

I'm assuming this is regex, so how about if you change to something like the following?
```
^(.*/)?pyproject.toml$
```

Reply:
This is great, I knew my implementation was simplistic, but I wasn't sure how to improve it.
Thank you 😄

Reply:
Glad that worked ☺️